balancer/rls/internal/balancer.go

```diff
@@ -75,18 +75,18 @@ func (lb *rlsBalancer) run() {
 // channel accordingly.
 // TODO(easwars): Handle updates to other fields in the service config.
 func (lb *rlsBalancer) handleClientConnUpdate(ccs *balancer.ClientConnState) {
-	grpclog.RLS.Infof("rls: service config: %+v", ccs.BalancerConfig)
```

Comment:
Same for RLS

Reply:
This has been replaced with an in-package defined "logger" var.